notificationworkerlambda/src/main/scala/com/gu/notifications/worker/TopicCounterLambda.scala

```diff
@@ -0,0 +1,52 @@
+package com.gu.notifications.worker
+
+import java.util.concurrent.{Executors, ScheduledExecutorService}
+
+import cats.effect.{ContextShift, IO}
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.auth.{AWSCredentialsProviderChain, DefaultAWSCredentialsProviderChain}
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.dynamodbv2.{AmazonDynamoDBAsync, AmazonDynamoDBAsyncClientBuilder}
+import com.amazonaws.services.s3.{AmazonS3, AmazonS3ClientBuilder}
+import com.gu.notifications.worker.utils.{Logging, TopicCountsS3}
+import db.{DatabaseConfig, RegistrationService}
+import doobie.util.transactor.Transactor
+import org.slf4j.{Logger, LoggerFactory}
+
+import scala.concurrent.{ExecutionContext, ExecutionContextExecutor}
+
+class TopicCounterLambda extends Logging {
+  def logger: Logger = LoggerFactory.getLogger(this.getClass)
+
+  def env = Env()
+
+  lazy val credentials: AWSCredentialsProviderChain = new AWSCredentialsProviderChain(
```

Comment:
There's a singleton if you want: `Aws.credentialsProvider`

Reply:
:+1: